crates/wasmtime/Cargo.toml

```diff
@@ -145,20 +147,22 @@ all-arch = [
   "wasmtime-winch?/all-arch",
 ]
 
+host-arch = ["wasmtime-cranelift?/host-arch"]
```

Comment:
Could you add a doc-block here and above `runtime` below indicating what the feature is?

Reply:
I added one for `runtime`. I think this feature isn't actually necessary - I modified the cranelift build so that if `all-arch` is enabled it doesn't complain if it can't build for the host arch.